README.md

```diff
@@ -5,7 +5,7 @@
 
 Running the command
 ```sh
-sudo pip install git+https://github.com/vokal/s3same.git
+sudo pip install s3same
 ```
 will install `s3same` (from the master branch of the source of this repo) as a command in `/usr/local/bin`.
```

Comment:
You probably want to note that it installs the latest stable release on PyPI instead of from the master branch.